### ISSUE TITLE
Minor Questions about Evaluation on ScanNet

### ISSUE THREAD

**Issue by ChushanZhang** [autonomousvision/monosdf#65]
Dear Authors, 

First and foremost, thank you for your exceptional work on this project!

I have a few questions regarding the evaluation on ScanNet, as I've noticed a minor difference between the results provided in your paper and those I obtained using your pre-trained model.

Interestingly, the F-score I computed is higher than the 0.733 value provided in your paper. However, the other metrics, such as Accuracy and Precision, are slightly worse than the ones you reported.

Would you kindly provide some guidance on whether I might have made an error in the process? I used your pre-trained model and ran the [evaluate.py](https://github.com/autonomousvision/monosdf/blob/main/scannet_eval/evaluate.py) script for evaluation.

Thank you for your time and assistance. I'm looking forward to your response.

The colum refers to：Acc    - Comp    - Prec    - Recall    - F-score
![ffeaeee978ff332a6345176fa31fe60](https://user-images.githubusercontent.com/36417130/229695691-4a0e7a9f-6727-4f63-9716-8662c50bb65b.png)

**Comment by niujinshuchong** [comment]
Hi, thanks for reporting it. The numbers in the paper are trained with eikonal loss weight 0.1 but the released models are trained with eikonal loss weight 0.05. So there are some minor differences.
